tools/version.go

```diff
@@ -0,0 +1,7 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package tools
+
+const DefaultBeatVersion = "8.5.0"
```

Comment:
I don't see it here. is there support for version qualifiers?

Reply:
updated the version qualifier